src/Xamarin.Android.Build.Tasks/Mono.Android/IntentFilterAttribute.Partial.cs

```diff
@@ -135,11 +143,13 @@ IEnumerable<XElement> GetData (string packageName)
 				.Concat ((DataPathPatterns ?? empty).Select (p => toData (toPathPattern, p)))
 				.Concat ((DataPathPrefixes ?? empty).Select (p => toData (toPathPrefix, p)))
 				.Concat ((DataPorts ?? empty).Select (p => toData (toPort, p)))
-				.Concat ((DataSchemes ?? empty).Select (p => toData (toScheme, p)));
+				.Concat ((DataSchemes ?? empty).Select (p => toData (toScheme, p)))
+				.Concat ((DataPathSuffixes ?? empty).Select (p => toData (toPathSuffix, p)))
```

Comment:
This class felt like maybe the code was trying to be too "clever", hindering readability. It would likely get rewritten to something a little more straightforward if/when we do this: https://github.com/xamarin/xamarin-android/issues/8272.